test/gtest/ucp/test_ucp_ep_reconfig.cc

```diff
@@ -243,4 +298,83 @@ UCS_TEST_P(test_ucp_ep_reconfig, basic)
     r_receiver->verify_configuration(*r_sender);
 }
 
-UCP_INSTANTIATE_TEST_CASE_TLS(test_ucp_ep_reconfig, rc, "rc");
+/* TODO: Remove skip condition after next PRs are merged. */
+UCS_TEST_SKIP_COND_P(test_ucp_ep_reconfig, basic,
+                     !has_transport("rc_x") || !has_transport("rc_v"))
+{
+    run(EXCLUDE_IFACES);
+}
+
+UCS_TEST_P(test_ucp_ep_reconfig, request_reset, "PROTO_REQUEST_RESET=y")
+{
+    if (is_single_transport()) {
+        /* One side will consume all ifaces and the other side will have no ifaces left to use */
+        UCS_TEST_SKIP_R("exclude_iface requires at least 2 transports to work "
+                        "(for example DC + SHM)");
+    }
+
+    if (has_transport("ib") && !has_resource(sender(), "rc_verbs")) {

```

Comment:
why checking for rc_verbs? can we have a more generic check?

Reply:
it is mandatory TL in IB, no reason to check all IB TLs

Reply:
why need to check devices?